lib/config/validation.js

```diff
@@ -182,7 +192,7 @@ async function validateConfig(config, isPreset, parentPath) {
               }
             }
             if (
-              selectors.includes(key) &&
+              (selectors.includes(key) || key === 'matchVersion') &&
```

Comment:
Why not just add it to the selectors array?

Reply:
Because that will lead to some problems. As it was noted in the issue's description, it shouldn't satisfy the "at least one selector" rule, and even if we set a condition to avoid that, in case of an error it will print out the whole `selectors` array as the required ones. https://github.com/renovateapp/renovate/blob/5215658b573eaf3c1c525df57af113be008b16ef/lib/config/validation.js#L149

Reply:
What makes this selector special that it is not included in the array while the other 5 are?